stubs/model.blade.php

```diff
@@ -2,6 +2,11 @@
 
 use Illuminate\Database\Eloquent\Model;
 use RonasIT\Support\Traits\ModelTrait;
+@if(!empty($importRelations))
+@foreach($importRelations as $value)
+use {{$value}};
+@endforeach
+@endif
```

Comment:
could we pass an empty array in case no imports required to avoid if here?

```suggestion
@foreach($importRelations as $value)
use {{$value}};
@endforeach
```